init/gravitino/init.sh

```diff
@@ -25,6 +25,10 @@ cp /root/gravitino/catalogs/jdbc-mysql/libs/mysql-connector-java-8.0.27.jar /roo
 cp /root/gravitino/catalogs/jdbc-postgresql/libs/postgresql-42.7.0.jar /root/gravitino/iceberg-rest-server/libs
 cp /root/gravitino/catalogs/jdbc-mysql/libs/mysql-connector-java-8.0.27.jar /root/gravitino/iceberg-rest-server/libs
 cp /tmp/gravitino/gravitino.conf /root/gravitino/conf
+
+# set env var 
+sed -i 's/__MYSQL_HOST_IP__/'"$MYSQL_HOST_IP"'/g' /root/gravitino/conf/gravitino.conf
+sed -i 's/__HIVE_HOST_IP__/'"$HIVE_HOST_IP"'/g' /root/gravitino/conf/gravitino.conf
```

Comment:
Normally, the configuration file gravitino.conf should be extracted as a configMap for frequent modifications.

Reply:
Actually, `gravitino.conf` is a config map.

https://github.com/apache/gravitino-playground/pull/56/files#diff-ce7b8c0869f3b377a49fd1d93471b53581e6cea8a0d66d016321a07085a5d54cR86-R94